includes/class-wcpdf-main.php

```diff
@@ -594,44 +594,25 @@ public function load_custom_and_local_fonts() {
 		require WPO_WCPDF()->plugin_path() . '/vendor/autoload.php';
 
 		$options = array(
-			'defaultFont'				=> 'dejavu sans',
-			'tempDir'					=> $this->get_tmp_path( 'dompdf' ),
-			'logOutputFile'				=> $this->get_tmp_path( 'dompdf' ) . "/log.htm",
-			'fontDir'					=> $this->get_custom_fonts_path(),
-			'fontCache'					=> $this->get_custom_fonts_path(),
-			'isRemoteEnabled'			=> true,
-			'isFontSubsettingEnabled'	=> true,
-			'isHtml5ParserEnabled'		=> true,
+			'defaultFont'             => 'dejavu sans',
+			'tempDir'                 => $this->get_tmp_path( 'dompdf' ),
+			'logOutputFile'           => $this->get_tmp_path( 'dompdf' ) . "/log.htm",
+			'fontDir'                 => $this->get_tmp_path( 'fonts' ),
+			'fontCache'               => $this->get_tmp_path( 'fonts' ),
+			'isRemoteEnabled'         => true,
+			'isFontSubsettingEnabled' => true,
+			'isHtml5ParserEnabled'    => true,
 		);
 
-		if ( ! @is_dir( $this->get_dompdf_fonts_path() ) || ! wp_is_writable( $this->get_dompdf_fonts_path() ) ) {
-			$rootDir = $this->get_custom_fonts_path();
-		}
-
-		// save custom fonts
-		$custom_fonts_options = new Options( $options );
-		if( isset( $rootDir ) ) {
-			$custom_fonts_options->setRootDir( $rootDir );
-		}
-		$dompdf = new Dompdf( $custom_fonts_options );
-		$dompdf->getFontMetrics()->saveFontFamilies();
-
-		// save local fonts
-		$options['fontDir']  = $options['fontCache'] = $this->get_tmp_path( 'fonts' );
-		$local_fonts_options = new Options( $options );
-		if( isset( $rootDir ) ) {
-			$local_fonts_options->setRootDir( $rootDir );
-		}
-		$dompdf->getFontMetrics()->setOptions( $local_fonts_options );
+		$rootDir     = $this->get_custom_fonts_path();
+		$options_obj = new Options( $options );
+		$options_obj->setRootDir( $rootDir );
```

Comment:
I didn't followed that because of this: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/c9e3489f548e186f5fd12002731908eec3a04bff/vendor/dompdf/dompdf/src/Options.php#L283

Reply:
Excellent! I didn't see that this is set independently 👍